### PR TITLE
Handle SAML reauth challenge.

### PR DESCRIPTION
**Pull request by postmasters**
`gsutil` can't really do much here so we display a message to ask users
to re-initialize credentials.

### CODE REVIEW

File: gslib/__main__.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
 # pylint: disable=g-bad-import-order
 import httplib2
 import oauth2client
-from google_reauth import reauth_creds
+from google_reauth import reauth_creds, errors as reauth_errors
```
Comment (NickGoog) [inline gslib/__main__.py:85]: Based on the surrounding style, I think these imports should be on separate lines

**Comment by postmasters** [comment]
Any update on the messaging, @NickGoog

**Comment by NickGoog** [comment]
If @starsandskies , hasn't said anything yet, I assume it's b/c there wasn't any issue.
Did the test suites pass for JSON PY2, XML PY2, JSON PY3, and XML PY3?

**Comment by starsandskies** [comment]
Yes, apologies - the wording of the error seems reasonable to me. Presumably, the only people receiving the error know what SAML IdP is?

**Comment by postmasters** [comment]
> If @starsandskies , hasn't said anything yet, I assume it's b/c there wasn't any issue.
> Did the test suites pass for JSON PY2, XML PY2, JSON PY3, and XML PY3?

Yes, of course. Though I did not run integration tests.

> Yes, apologies - the wording of the error seems reasonable to me. Presumably, the only people receiving the error know what SAML IdP is?

Presumably yes. Or their IT Admin folks should know.

**Comment by postmasters** [comment]
Quick ping, folks. Thanks!

**Pull request merged**